projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -604,8 +608,8 @@ export interface ExplorerGeneratedDashboard {
   onReady(dashboard: Dashboard): void;
 }
 
-interface ModelSpecificationJson extends ModelJson {
-  value?:
+export interface ModelSpecificationJson extends ModelJson {
```

Comment:
If you're exporting this, need a clearer name (e.g. SpecificationBackedColumnModelJson - I believe that's we call this form of column)

Reply:
reanmed